mmdeploy/codebase/mmpose/deploy/pose_detection_model.py

```diff
@@ -159,6 +159,84 @@ def pack_result(self,
         return data_samples
 
 
+@__BACKEND_MODEL.register_module('yolox-pose_end2end')
+class YoloXPoseEnd2EndModel(End2EndModel):
```

Comment:
so do we need to add configs for yolox, just like simcc?

Reply:
> so do we need to add configs for yolox, just like simcc?

Yes, `yolox-pose_onnxruntime_static.py` and `yolox-pose_tensorrt_static-640x640.py` are added.